src/main/java/io/cdap/plugin/http/sink/batch/HTTPSinkConfig.java

```diff
@@ -24,16 +24,22 @@
 import io.cdap.cdap.api.annotation.Name;
 import io.cdap.cdap.api.data.schema.Schema;
 import io.cdap.cdap.etl.api.FailureCollector;
+import io.cdap.cdap.etl.api.validation.InvalidConfigPropertyException;
 import io.cdap.cdap.format.StructuredRecordStringConverter;
 import io.cdap.plugin.common.ReferenceNames;
 import io.cdap.plugin.common.ReferencePluginConfig;
 
+import io.cdap.plugin.http.source.batch.HttpBatchSourceConfig;
+import io.cdap.plugin.http.source.common.http.AuthType;
```

Comment:
It seems like the common classes are present in the Http source package, Can you refactor those into a common package ?

Reply:
.* imports are not allowed